app/views/users/getting_started.mobile.haml

```diff
@@ -59,10 +59,6 @@
       %fieldset
         = label_tag 'profile[first_name]', t('profiles.edit.your_name'), class: "bootstrapped"
         = text_field_tag 'profile[first_name]', current_user.first_name
-        = image_tag 'ajax-loader2.gif', id: "gs-name-form-spinner", class: "hidden"
-        %span.saved{class: "hidden"}
-          = image_tag "mobile/check_yes_ok.png"
-          = t(".saved")
```

Comment:
This was introduced with https://github.com/diaspora/diaspora/commit/507e8ee1999 but there is no javascript for that page so there is no way to show these elements.